assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentToolUtils.java

```diff
@@ -581,7 +581,9 @@ else if ("remove".equals(addUpdateRemoveAssignment)) {
                                             if (grade != null && gradingService.isUserAbleToGradeItemForStudent(gradebookUid, associateGradebookAssignmentId, submitterId)) {
                                                 gradingService.setAssignmentScoreString(gradebookUid, associateGradebookAssignmentId, submitterId, grade, "");
                                                 String comment = StringUtils.isNotEmpty(cm.get(submitterId)) ? cm.get(submitterId) : "";
-                                                gradingService.setAssignmentScoreComment(gradebookUid, associateGradebookAssignmentId, submitterId, comment);
+                                                if (! StringUtils.isBlank(comment)) {
```

Comment:
More readable:

```suggestion
                                                if (StringUtils.isNotBlank(comment)) {
```

Reply:
@bjones86 - Thanks. I'll soon force push an update including this change.